breathecode/authenticate/models.py

```diff
@@ -431,6 +433,14 @@ def __init__(self, *args, **kwargs):
                                  blank=True)
     cohort = models.ForeignKey(Cohort, on_delete=models.CASCADE, null=True, default=None, blank=True)
     role = models.ForeignKey(Role, on_delete=models.CASCADE, null=True, default=None, blank=True)
+    event_slug = models.SlugField(max_length=40,
```

Comment:
These info can change and make this data useless, this is unacceptable in a relational context